### ISSUE TITLE
Use [cypress-accessibility-checker](https://www.npmjs.com/package/cypress-accessibility-checker) instead of `karma-accessibility-checker`

### ISSUE THREAD

**Issue by kennylam** [carbon-design-system/carbon-for-ibm-dotcom#10377]

**Comment by m4olivei** [comment]
Stumbled on this one. Just out of curiosity, will the replacement proposed here be the ultimate resolution for the persistant `ci-check / a11y` job failures? If a resolution here is far off, would there be any interest in attempting to fix the `ci-check / a11y` job run in the interim in an effort to cut down on the number of failed jobs? e.g. https://github.com/carbon-design-system/carbon-for-ibm-dotcom/actions/workflows/ci-check.yml?query=branch%3Amain

It's been failing, recently at least, b/c of a dependency issue with the `exceljs` package.
